tools/deployment/onnx2tensorrt.py

```diff
@@ -170,58 +149,92 @@ def parse_args():
         nargs='+',
         default=[58.395, 57.12, 57.375],
         help='Variance value used for preprocess input data')
+    parser.add_argument(
+        '--min-shape',
+        type=int,
+        nargs='+',
+        default=None,
+        help='Minimum input size of the model in TensorRT')
+    parser.add_argument(
+        '--max-shape',
+        type=int,
+        nargs='+',
+        default=None,
+        help='Maximum input size of the model in TensorRT')
     parser.add_argument(
         '--workspace-size',
         type=int,
         default=1,
         help='Max workspace size in GiB')
+
     args = parser.parse_args()
     return args
 
 
+def parse_normalize_cfg(test_pipeline):
+    transforms = test_pipeline[1]['transforms']
+    norm_config_li = [_ for _ in transforms if _['type'] == 'Normalize']
+    assert len(norm_config_li) == 1
+    norm_config = norm_config_li[0]
+    return norm_config
+
+
 if __name__ == '__main__':
 
     assert is_tensorrt_plugin_loaded(), 'TensorRT plugin should be compiled.'
     args = parse_args()
-
+    warnings.warn(
+        'Arguments like `--to-rgb`, `--mean`, `--std`, `--dataset` are '
+        'deprecated and will be removed in future releases.')
```

Comment:
Add some annotations in the warning message about where these args are moved to.